lib/gitlib.sh

```diff
@@ -393,8 +393,10 @@ gitlib::update_release_issue () {
       logecho "Created issue #$issue_number on github:"
       logecho "http://github.com/$repo/issues/$issue_number"
     else
-      logecho "There was a problem creating the release tracking issue"
-      logecho "This should be done manually."
+      logecho "$WARNING: There was a problem creating the release tracking" \
```

Comment:
There should be a space between "tracking" and "issue" right? I forget if `\` auto-adds a space.

Reply:
bash conveniently compresses 1 or more spaces into 1.

Reply:
Doh, I missed that there was a space between the `"` and `\`. LGTM